### PR TITLE
CRDCDH-1782 Add Test MDF

### PR DESCRIPTION
**Pull request by amattu2**
### Overview

This PR adds the Test MDF to our CRDC-DH repo.

### Change Details (Specifics)

- Add the single YML file from https://github.com/CBIIT/bento-mdf/blob/main/python/tests/samples/crdc_datahub_mdf.yml
- FE/BE do not support a single YAML file ATM, so the mdf file is referenced twice for the `model-file` and `prop-file` fields
- No loading files were provided, we can add it later

### Related Ticket(s)

CRDCDH-1695

### CODE REVIEW

File: cache/content.json [hunk 1]
```diff
@@ -152,5 +152,32 @@
...
+        "subject": "Participant",
+        "specimen": "Sample",
+        "data_file": "File"
+      },
+      "file-nodes": {
+        "data_file": {
+          "id-field": "file_id",
+          "name-field": "",
+          "size-field": "file_size",
+          "md5-field": ""
```
Comment (amattu2) [inline cache/content.json:174]: I didn't see an obvious md5-field property, so I left it blank.

File: cache/content.json [hunk 2]
```diff
@@ -152,5 +152,32 @@
...
+        "program": "Program",
+        "study": "Study",
+        "subject": "Participant",
+        "specimen": "Sample",
+        "data_file": "File"
+      },
+      "file-nodes": {
+        "data_file": {
+          "id-field": "file_id",
+          "name-field": "",
```
Comment (amattu2) [inline cache/content.json:172]: Same for this. Unsure which property it would be

File: cache/content.json [hunk 3]
```diff
@@ -152,5 +152,32 @@
...
       "1.9.1"
     ]
+  },
+  "Test MDF": {
+    "model-file": "crdc_datahub_mdf.yaml",
+    "prop-file": "crdc_datahub_mdf.yaml",
+    "readme-file": "README.md",
+    "loading-file": "",
+    "semantics": {
+      "main-nodes": {
```
Comment (amattu2) [inline cache/content.json:162]: I removed the PI semantic as I don't see one in the model. I might have missed it though.

File: cache/content.json [hunk 4]
```diff
@@ -152,5 +152,32 @@
...
     ]
+  },
+  "Test MDF": {
+    "model-file": "crdc_datahub_mdf.yml",
+    "prop-file": "crdc_datahub_mdf.yml",
+    "readme-file": "README.md",
+    "loading-file": "",
+    "semantics": {
+      "main-nodes": {
+        "program": "Program",
```
Comment (n2iw) [inline cache/content.json:163]: program should be removed also, I don't see it in the model.

File: cache/content.json [hunk 5]
```diff
@@ -152,5 +152,32 @@
...
+        "study": "Study",
+        "subject": "Participant",
```
Comment (n2iw) [inline cache/content.json:165]: "subject" should be "participant"

File: cache/content.json [hunk 6]
```diff
@@ -152,5 +152,32 @@
...
+        "specimen": "Sample",
```
Comment (n2iw) [inline cache/content.json:166]: "specimen" should be "sample"

File: cache/content.json [hunk 7]
```diff
@@ -152,5 +152,32 @@
...
+        "data_file": "File"
```
Comment (n2iw) [inline cache/content.json:167]: "data_file" should be "file"

File: cache/content.json [hunk 8]
```diff
@@ -152,5 +152,32 @@
...
+      },
+      "file-nodes": {
+        "data_file": {
```
Comment (n2iw) [inline cache/content.json:170]: "data_file" should be "file"

File: cache/content.json (hunk 2 above)
Comment (n2iw) [reply cache/content.json:172]: I'll ask Nelson to add missing properties.

**Review by n2iw — CHANGES_REQUESTED** [review]

File: cache/content.json (hunk 8 above)
Comment (amattu2) [reply cache/content.json]: Updated in b9e57f7

File: cache/content.json (hunk 7 above)
Comment (amattu2) [reply cache/content.json]: Updated in b9e57f7

File: cache/content.json (hunk 6 above)
Comment (amattu2) [reply cache/content.json]: Updated in b9e57f7

File: cache/content.json (hunk 5 above)
Comment (amattu2) [reply cache/content.json]: Updated in b9e57f7

File: cache/content.json (hunk 4 above)
Comment (amattu2) [reply cache/content.json]: Removed in b9e57f7

File: cache/content.json (hunk 2 above)
Comment (amattu2) [reply cache/content.json]: Updated MDF and defined the name/md5 fields in the content config.

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**